.ci/pitest-survival-check-xml.groovy

```diff
@@ -293,13 +293,13 @@ private static int printComparisonToConsole(Set<Mutation> survivingMutations,
 }
 
 /**
- * Whether a set has only stable mutations.
+ * Whether a set has only unstable mutations.
  *
  * @param mutations A set of mutations
- * @return {@code true} if a set has only stable mutations
+ * @return {@code true} if a set has only unstable mutations
  */
-private static boolean hasOnlyStableMutations(Set<Mutation> mutations) {
-    return mutations.every { !it.isUnstable() }
+private static boolean hasOnlyUnstableMutations(Set<Mutation> mutations) {
+    return mutations.every { it.isUnstable() }
```

Comment:
The problem was that `!hasOnlyStableMutations != hasOnlyUnstableMutations`, `!hasOnlyStableMutations` could mean that it has both stable and unstable and having unstable made `hasOnlyStableMutations` return false, which made `!hasOnlyStableMutations` true. BUT this does not mean that it only has unstable, it could be a mix of both.